libtransmission/rpcimpl.cc

```diff
@@ -1226,6 +1230,16 @@ char const* torrentSet(tr_session* session, tr_variant* args_in, tr_variant* /*a
             tr_torrentSetSpeedLimit_KBps(tor, TR_DOWN, tmp);
         }
 
+        if (auto val = bool{}; tr_variantDictFindBool(args_in, TR_KEY_sequentialDownload, &val))
+        {
+            tor->setSequentialDownload();
+        }
+
+        if (auto val = bool{}; tr_variantDictFindBool(args_in, TR_KEY_randomDownload, &val))
+        {
+            tor->setRandomDownload();
+        }
```

Comment:
RPC changes need to be documented in `docs/rpc-spec.md`.

In this case you'd want to 

- add `sequentialDownload` to section 3.2 (torrent-set)
- add `sequentialDownload` to section 3.3 (torrent-get)
- and `sequentialDownload` to section 5 (API changelog).

Since this adds new functionality, we'd want to bump `rpc-version-semver` to 5.4.0 and `rpc-version` to 18, which needs to be bumped in

- `docs/rpc-spec.md` section 5 (changelog)
- [`RpcVersion` in rpcimpl.cc](https://github.com/transmission/transmission/blob/44546674d955c9b2982ec3c2b18d547142b0c9e4/libtransmission/rpcimpl.cc#L50)
- [`RpcVersionSemver` in rpcimpl.cc](https://github.com/transmission/transmission/blob/44546674d955c9b2982ec3c2b18d547142b0c9e4/libtransmission/rpcimpl.cc#L52)

I know that looks like a lot but it's not too terrible; a bunch of one-line changes :smile_cat:

Reply:
Done - I also had to tag `Transmission 4.0.1` in `rpc-spec.md` in the changelog to set the new versions of the RPC - but feel free to amend the PR to set it to a different version :+1: